src/Microsoft.AspNet.Authentication/AuthenticationHandler.cs

```diff
@@ -139,7 +141,7 @@ protected virtual Task FinishResponseAsync()
 
         private async Task HandleAutomaticChallengeIfNeeded()
         {
-            if (!ChallengeCalled && Options.AutomaticAuthentication && Response.StatusCode == 401)
+            if (!ChallengeCalled && Options.AutomaticAuthenticate && Response.StatusCode == 401)
```

Comment:
Good one, uncovered a lot of tests using the old one too since they all got renamed together :(